app/services/api/training_periods/teacher_status.rb

```diff
@@ -0,0 +1,43 @@
+module API::TrainingPeriods
+  class TeacherStatus
+    attr_reader :training_period, :teacher
+
+    delegate :started_on, :finished_on, :trainee, to: :training_period, private: true
```

Comment:
I don't think we're using `trainee` anywhere; maybe we could make the interface simpler by just accepting the `latest_training_period` and accessing `latest_training_period.trainee.teacher` as I think we preload this in the query anyway?

Reply:
Ah yes that's right; we used to include `trainee.teacher` but then changed the serializer to pass the `teacher` down as well as `training_period`, fine to keep `teacher` in the parameters, but I think we can still drop the `trainee` delegate here